src/main/webapp/dashboard-manager.js

```diff
@@ -0,0 +1,142 @@
+class DashboardManager {
+  constructor() {
+    this.data = this.initData();
+
+    this.dashboard = new google.visualization.Dashboard(document.getElementById('dashboard'));
+    this.aggregationSelector = createNewAggregationSelector();
+    this.table = createNewTable();
+    this.pieChart = createNewPieChart();
+
+    google.visualization.events.addListener(
+        this.aggregationSelector, 'statechange', this.updateAndDrawData.bind(this));
+  }
+
+  /* Get the initial data to be shown to the user and populate the main datatable*/
+  initData() {
+    let data = new google.visualization.DataTable();
+    data.addColumn('string', 'Serial Number');
+    data.addColumn('string', 'Status');
+    data.addColumn('string', 'Asset ID');
+    data.addColumn('string', 'User');
+    data.addColumn('string', 'Location');
+
+    data.addRow(['SN12345', 'Provisioned', '1e76c3', 'James', 'Texas']);
```

Comment:
Please comment that these are test data and consider adding a TODO here that replaces the test data with real data.

Reply:
added